caps/pip.md

```diff
@@ -0,0 +1,431 @@
+# Parity Light Protocol (PIP)
+
+The Parity Light Protocol is a variation of LES designed and implemented by Parity Tech for the Parity Ethereum client.
+
+Please refer to the LES specification for information on the purpose of light clients and protocols.
+
+PIP adopts a flow-control mechanism closely analogous to a [token-bucket rate limiter](https://en.wikipedia.org/wiki/Token_bucket) where the client is expected to mirror the server token-bucket state (as exceeding the 'burstiness' depth is a violation that results in disconnection). This is also explained in more detail in the [LES][LES] documentation.
+
+PIP utilises `Canonical Hash Tries` (CHTs), which are also described in the [LES][LES] documentation.

```

Comment:
For "Canonical Hash Tries", it would be good to link to the related section. How to do it:

Add a link definition to the bottom of the document:

```text
[Canonical Hash Tries]: ./les.md#canonical-hash-tries
```

Then replace the backquotes with `[` and `]`:

```text
[Canonical Hash Tries]
```